src/Auth0.ManagementApi/Models/ClientResourceServerAssociation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Auth0.ManagementApi.Models
+{
+    public class ClientResourceServerAssociation
+    {
+        /// <summary>
+        /// The resource server identifier.
+        /// </summary>
+        [JsonProperty("identifier")]
+        public string Identifier { get; set; }
+
+        /// <summary>
+        /// The scopes to request from the user to access the resource server.
```

Comment:
I'd say it's more like "The scopes granted to the client to access the resource server."